### DISCUSSION TITLE
Web Server Buildpack Maintainers

### DISCUSSION THREAD

**Discussion by ryanmoran** [paketo-buildpacks/feedback#6] [Q&A] [answered]
Hi folks! We are going to be starting the process of implementing the Web Servers Buildpack RFC. This will mean that we want to create a maintainer team to handle maintenance of the buildpacks as they move through incubation in the `paketo-community` org.

If you are interested in becoming a maintainer for the Web Servers buildpacks, please leave a comment on this topic and let us know. We will hopefully be able to find someone by the next working group meeting on 2/2/21.

**Comment by arjun024** [comment, marked as answer]
Sounds great! Interested.

**Reply by ryanmoran** [reply]
@thitch97 is also stepping up to maintain this buildpack.
